src/Controls/tests/DeviceTests/ControlsDeviceTestExtensions.cs

```diff
@@ -13,6 +14,31 @@ namespace Microsoft.Maui.DeviceTests
 {
 	public static class ControlsDeviceTestExtensions
 	{
+
+		public static bool IsAccessibilityElement(this VisualElement view)
+		{
+			bool result = AssertionExtensions.IsAccessibilityElement(view);
+
+#if IOS
+			// UIView.IsAccessibilityElement will never be true if VoiceOver isn't running
+			//
+			// Because the 'AssertionExtensions.IsAccessibilityElement' just fakes the actual values
+			// when VO isn't running we can't really test that calling `SetIsInAccessibleTree` with
+			// false works unless we turn on VO
```

Comment:
I think you opened an issue for this. Could you link to the issue in the comment here?

Reply:
Are you thinking of this issue?
https://github.com/dotnet/maui/issues/11762

It's not really related. This comment just explains why I have all this odd-looking code here and why none of this is really that accurate unless you run these on a physical device